docs/source/asr/datasets.rst

```diff
@@ -190,3 +192,81 @@ in the same directory as the manifest, or even in any specific directory structu
 
 Once you have a manifest that describes each audio file in your dataset, you can then use the dataset by passing
 in the manifest file path in your experiment config file, e.g. as `training_ds.manifest_filepath=<path/to/manifest,json>`.
+
+Tarred Datasets
+---------------
+
+If you are running experiments on a cluster with datasets stored on a distributed file system, you will likely
+want to avoid constantly reading many small files and would prefer tarring your audio files.
+There are tarred versions of some NeMo ASR Dataset classes for this case, such as the ``TarredAudioToCharDataset``
+(corresponding to the ``AudioToCharDataset``) and the ``TarredAudioToBPEDataset`` (corresponding to the
+``AudioToBPEDataset``).
+The tarred audio dataset classes in NeMo use `WebDataset <https://github.com/tmbdev/webdataset>`_.
+
+To use an existing tarred dataset instead of a non-tarred dataset, you will need to set ``is_tarred: true`` in
+your experiment config file.
+Then, you will need to pass in the paths to all of your audio tarballs in ``tarred_audio_filepaths``, either as a list
+of filepaths (e.g. ``['/data/shard1.tar', '/data/shard2.tar']``), or in a single brace-expandable string (e.g.
+``'/data/shard_{1..64}.tar'``).
+
+.. note::
+  For brace expansion, there may be cases where ``{x..y}`` syntax cannot be used due to shell interference.
+  This occurs most commonly inside SLURM scripts. Therefore we provide a few equivalent replacements.
+  Supported opening braces (equivalent to ``{``) are ``(``, ``[``, ``<`` and the special tag ``_OP_``.
+  Supported closing braces (equivalent to ``}``) are ``)``, ``]``, ``>`` and the special tag ``_CL_``.
+  For SLURM based tasks, we suggest the use of the special tags for ease of use.
```

Comment:
Lets guide users to user the `_OP_` and `_CL_` tags - they are guarenteed to work. The others may or may not work depending on the shell they use (bash vs zsh has different reactions to < vs [ )

Reply:
Sounds good to me, will update this.